### PR TITLE
[TEST] use test data as mockdata

### PR DESCRIPTION
**Pull request by jackstar12**
the `mock_data.zip` is and the migration workflow seems to fail randomly with mismatching db versions.

generating the mock data through a script makes sure the db is up to date

### CODE REVIEW

**Review by dni — APPROVED** [review]

File: lnbits/commands.py [hunk 1]
```diff
@@ -104,6 +113,27 @@ async def db_versions():
         return await get_dbversions(conn)
 
 
+@db.command("mock")
+def database_mock():
```
Comment (motorina0) [inline lnbits/commands.py:117]: - still want to keep this?

File: Makefile [hunk 2]
```diff
@@ -48,18 +48,16 @@ test-real-wallet:
...
 
 test-migration:
-	rm -rf ./migration-data
-	mkdir -p ./migration-data
-	unzip tests/data/mock_data.zip -d ./migration-data
+	make test
 	HOST=0.0.0.0 \
 	PORT=5002 \
-	LNBITS_DATA_FOLDER="./migration-data" \
+	LNBITS_DATA_FOLDER="./tests/data" \
```
Comment (motorina0) [inline Makefile:54]: - are we sure there is no clean-up hook that cleans up the data?

File: lnbits/commands.py (hunk 1 above)
Comment (jackstar12) [reply lnbits/commands.py:117]: i dont think so

**Comment by jackstar12** [comment]
removed the script, https://github.com/lnbits/lnbits/pull/1929/commits/0c69332fe77dbcb2009c18965850a928304202d6 could be its own pr now, but I think we can leave it here

**Review by arcbtc — APPROVED** [review]

**Pull request merged**